static-assets/components/cstudio-forms/controls/file-name.js

```diff
@@ -134,7 +134,7 @@ YAHOO.extend(CStudioForms.Controls.FileName, CStudioForms.CStudioFormField, {
   _onChange: function (evt, obj) {
     var oldValue = obj.value;
     obj.value = obj.inputEl.value;
-    if (obj.value != '' && oldValue != obj.value)
+    if (oldValue != obj.value)
```

Comment:
The `not empty` validation was removed because the validation cleans the error when the field is empty. With the validation there was an scenario where the error was not being cleared properly.

Reply:
So what happens now when it is empty?

Reply:
When the value is empty, it doesn't verify if the path exists, it only clears any previous `path exists` error